internal/cli/docs/generate-cli-docs.go

```diff
@@ -173,12 +173,19 @@ func writeFileHeader(w io.Writer, sidebarLabel string) error {
 	if sidebarLabel == "epinio" {
 		sidebarLabel = "epinio cli"
 	}
+	description := sidebarLabel
+	keywords := sidebarLabel
```

Comment:
```suggestion
```

Reply:
I wasn't sure about how to do that but also thought I'd leave them duplicated in case in the future I wanted to manipulate them independently, or pass them into the function. But, probably not.